queries.sql

```diff
@@ -31,3 +31,43 @@ WHERE NAME <> 'Gabumon';
 SELECT *
 FROM animals
 WHERE weight_kg BETWEEN 10.4 AND 17.3;  
+
+
+BEGIN;
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+COMMIT;
+
+
+BEGIN;
+DELETE
+FROM animals;
+ROLLBACK;
+
+
+BEGIN;
+DELETE
+FROM animals
+WHERE date_of_birth > 'January 1, 2022';
+SAVEPOINT younger_deleted;
+UPDATE animals
+SET weight_kg = - (weight_kg);
+ROLLBACK TO younger_deleted;
```

Comment:
- [ ] after this `ROLLBACK` you should UPDATE all animals' weights that are negative to be their weight multiplied by -1.